code-of-conduct.md

```diff
@@ -0,0 +1,40 @@
+# Dat Ecosystem - Code of Conduct
+
+Like the technical community as a whole, the [Dat Ecosystem](./MANIFESTO.md) is made up of a mixture of professionals and volunteers from all over the world, working on every aspect of the mission - including mentorship, teaching, and connecting people.
+
+Diversity is one of our huge strengths, but it can also lead to communication issues and unhappiness. To that end, we have a few ground rules that we ask people to adhere to.
+
+This code applies to anyone interacting in community spaces equally, including consortium members.
+
+This isn’t an exhaustive list of things that you can’t do. Rather, take it in the spirit in which it’s intended - a guide to make it easier to enrich all of us and the technical communities in which we participate.
+
+This code of conduct applies to all spaces (real or digital) managed by the Dat Ecosystem group. This includes but is not limited to the Discord Chat, IRC, mailing lists, the issue trackers, meeting rooms (zoom), Youtube channels and any other system created or maintained by the group.
+
+In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+
+If you believe someone is violating the code of conduct, we ask that you report it using [our private contact system](https://dat-ecosystem.org/organization).
+
+For more details please see our [Reporting Guidelines](./code-of-conduct-contact.md).
```

Comment:
ok, so i could probably live with a version that removes the following to paragraphs:

> 1. What if your report concerns a possible violation by a consortium member?
> 2. What happens after you file a report?

To me they feel like too much of an involved process with fine grained rules and even time guarantees and promise of "high priority" and stuff like that.

**and instead replaces them with a line that says something like:**

*"we will do the best we can and use our consciouss and try to be lovely people and good people and be fair and find a way to resolve it...."*  or something in that spirit.



I would prefer to first have our first "incident" (hopefully never) and then handle it by using common sense and our best intentions and - if thats not enough - i would then maybe start crafting a more detailed document based on our actual experience.

And once we have an actual real world "incident" (hopefully never) we handle it with best intentions and common sense and if there are problems, then use that experience to add guarantees or process details or whatever.

Reply:
Okay, I will propose some changes to the reporting guide within a day.

Reply:
Merged #4 is it okay to you now?